Sources/Protocols/StoryboardType.swift

```diff
@@ -16,17 +16,18 @@ public protocol StoryboardType {
 public struct StoryboardReference<S: StoryboardType, T> {
 
     public let id: String
+    public let bundle: Bundle?
```

Comment:
I just noticed this small thing. These properties do not need to be public, doesn't they?